buildtrigger/test/test_githosthandler.py

```diff
@@ -93,9 +93,9 @@ def test_list_build_source_namespaces():
     ]
 
     found = get_bitbucket_trigger().list_build_source_namespaces()
-    found.sort()
+    sorted(found, key=lambda d: sorted(d.items()))
```

Comment:
This isn't actually going to sort anything, since it *returns* the sorted value

Reply:
Fixed.